.gitignore

```diff
@@ -21,7 +21,7 @@ project.xml.log
 *.files
 
 # CMake
-CMakeLists.txt
+#CMakeLists.txt
```

Comment:
No.
`CMakeLists.txt` has to be generated by the build system generator and thus (as a generated file) is included in the gitignore.

Reply:
No!!!. If you download a library from somewhere it's very easy to implement it you project with cmake. You just forbid that, so why use cmake at all???. Wait for the examples, they are more illustrative. I'm demonstrating how to use a header only and a static library.